lib/ansible/modules/system/syspatch.py

```diff
@@ -69,7 +79,7 @@
   returned: always
   type: str
   sample: "syspatch: need root privileges"
-reboot_needed:
+reboot_required:
```

Comment:
Keep `reboot_needed` for backward compatability. Since users might be using it.

Reply:
Sounds good. This was changed before the 2.9 release, but now that it's out I think it makes sense to keep it as is